packages/firestore/src/remote/watch_change.ts

```diff
@@ -449,17 +454,18 @@ export class WatchChangeAggregator {
       hashCount = 0
     } = unchangedNames;
 
-    // TODO(Mila): Remove this validation, add try catch to normalizeByteString.
-    if (typeof bitmap === 'string') {
-      const isValidBitmap = this.isValidBase64String(bitmap);
-      if (!isValidBitmap) {
-        logWarn('Invalid base64 string. Applying bloom filter failed.');
-        return false;
+    let normalizedBitmap: Uint8Array;
+    try {
+      normalizedBitmap = normalizeByteString(bitmap).toUint8Array();
+    } catch (err) {
+      if (err instanceof Base64DecodeError) {
+        logWarn('Base64 string error: ', err);
```

Comment:
Please make this warning message more detailed. This warning will be shown in the log without any context so it needs to explain what it means and what the effects are.

For example, something like 

```
Decoding base64 bloom filter in existence filter failed (${err.message}); ignoring bloom filter and falling back to full requery.
```